chef/cookbooks/cpe_chrome/resources/cpe_chrome.rb

```diff
@@ -94,5 +96,31 @@ def manage_chrome_osx
         ],
       }
     end
+
+    unless node['cpe_chrome']['mp']['UseMasterPreferences'] == false
+      mprefs = node['cpe_chrome']['mp']['FileContents'].reject { |v| v.nil? }
```

Comment:
ah actually hold on you have something in the comments that shows usage. I can use that to give you an example. I assume you just want all keys with `nil` values dropped?? It is a bit confusing what your intent is because you initialize it to an array in the attributes file, but in your example it is a hash.